packages/cozy-client/src/Query.jsx

```
@@ -88,92 +101,52 @@ const computeChildrenArgs = queryAttributes => {
-      if (
-        this.props.fetchPolicy &&
-        typeof this.props.fetchPolicy === 'function' &&
-        this.props.fetchPolicy(queryState)
-      ) {
-        fetchQuery(this.client, this.observableQuery)
-      }
-    } else {
-      fetchQuery(this.client, this.observableQuery)
-    }
+/**
+ * @param {object} props
```

ESLint

[error] Missing JSDoc `@param` "props" description.

(jsdoc/require-param-description)